frontend/src/BlogCard.tsx

```diff
@@ -12,22 +12,28 @@ export default function BlogCard(props: Readonly<BlogCardProps>) {
     const [description, setDescription] = useState(props.blog.description);
     const [isEditing, setIsEditing] = useState(false);
 
+
     function deleteThisItem() {
         axios.delete("/api/blog/" + props.blog.id)
             .then(props.onBlogItemChange)
-            .catch(error => {
-                console.error('There was an error deleting the blog!', error);
-            });
     }
 
-    function changeText(event: React.ChangeEvent<HTMLInputElement>): void {
-        setDescription(event.target.value);
+
+    function changeText(event: React.ChangeEvent<HTMLInputElement>): void{
+        const newDescription  = event.target.value
+        setDescription(newDescription);
+        axios.put("/api/blog/" + props.blog.id,{
+            ...props.blog,
+            description: newDescription
+
+        } as Blog)
+
     }
```

Comment:
You are firing a PUT request on every keystroke the user does on your field. This may put a heavy load on your server. You could think about implementing a debounce technique to minimize the number of requests.